### PR TITLE
chore: add outline to string literals

### LINKED ISSUE #797: Design: Striped table has same background as string literals
String literals have a light grey background that is the same as the striped table background 

@Shecki  is this tolerable or do we need to adjust? :
image

### PR DESCRIPTION
**Pull request by TimonRey**
closes #797 

@Shecki link to an example: https://commercetools-docs-kit-git-add-outline-to-string-literals.commercetools.vercel.app/docs-smoke-test/views/links

### CODE REVIEW

**Review by Shecki — CHANGES_REQUESTED** [review]
@TimonRey  it now looks like it has two outlines?
The old one should be replaced with the new color.

Also the corner-radius should be used from the old outline.

Thanks

**Comment by emmenko** [comment]
@Shecki is this only relevant for inline code elements inside tables? Or in general?

**Comment by Shecki** [comment]
@emmenko in gerneral

**Comment by emmenko** [comment]
Ok

File: packages/ui-kit/src/components/markdown.js [hunk 1]
```diff
@@ -220,6 +220,7 @@ const TableHeader = styled.th`
 `;
 
 const InlineCode = styled.code`
+  outline: 1px solid ${colors.light.borderSecondary};
```
Comment (emmenko) [inline packages/ui-kit/src/components/markdown.js:223]: You shouldn't change the `outline` style, but the border color instead.

**Review by emmenko — CHANGES_REQUESTED** [review]

File: packages/ui-kit/src/components/markdown.js [hunk 2]
```diff
@@ -222,6 +222,7 @@ const TableHeader = styled.th`
 const InlineCode = styled.code`
   background-color: ${colors.light.surfaceInlineCode};
   border: 1px solid ${colors.light.surfaceInfo};
+  border-color: ${colors.light.borderSecondary};
```
Comment (emmenko) [inline packages/ui-kit/src/components/markdown.js:225]:
```suggestion
  border: 1px solid ${colors.light.borderSecondary};
```

**Review by emmenko — CHANGES_REQUESTED** [review]

**Review by emmenko — APPROVED** [review]

**Comment by emmenko** [comment]
VRTs approved ✅

**Comment by Shecki** [comment]
@TimonRey  can we do something about this view?
image

That it looks like A or B?
image

**Comment by Shecki** [comment]
**Please don't look at the colors here**..
It's just about the form of the background

**Comment by emmenko** [comment]
Option B is not possible.

Option A also does not really make sense because it's essentially a sentence in a new line.

**Pull request merged**